clients/web/test/spec/setApiSpec.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Start the girder backbone app.
+ */
+girderTest.startApp();
+
+describe('Test setApiRoot() and setStaticRoot() functions', function () {
+    it('Check for default values and mutation', function () {
+        waitsFor(function () {
+            return $('.g-frontpage-body').length > 0;
+        });
+
+        runs(function () {
+            var host = 'http://localhost:30019';
```

Comment:
I don't think you need to know it.  Just `expect(girder.rest.apiRoot.slice(girder.rest.apiRoot.indexOf('/', 7))).toBe('/api/v1')`